src/reanimated2/animations.js

```diff
@@ -161,7 +161,6 @@ export function withSpring(toValue, userConfig, callback) {
       animation.velocity = previousAnimation.velocity || 0;
```

Comment:
I think we also have a bug here.  Probably it should be: 
`previousAnimation.velocity || animation.velocity || 0;`

Reply:
I think this makes sense